qtconsole/tests/test_00_console_widget.py

```diff
@@ -448,7 +486,7 @@ def test_prompt_cursors(self):
                          w._prompt_pos - len(w._prompt))
 
         # insert some text before the prompt
-        w._append_plain_text('line', before_prompt=True)
+        w._append_plain_text('line', before_prompt = True)
```

Comment:
Same as above, I think we could revert this change since the equal is for a `kwarg`.  So something like:
```suggestion
        w._append_plain_text('line', before_prompt=True)
```

Reply:
So the idea is kwarg equals doesn't have space? Changed.